pkg/config/config.go

```diff
@@ -65,6 +67,11 @@ func (cfg *Config) BindFlags() {
 		"Configures the logger to use a Zap development config (encoder=consoleEncoder,logLevel=Debug,stackTraceLevel=Warn, no sampling), "+
 			"otherwise a Zap production config will be used (encoder=jsonEncoder,logLevel=Info,stackTraceLevel=Error), sampling).",
 	)
+	flag.BoolVar(
+		&cfg.EnableDefaultTagging, flagEnableDefaultTagging,
+		true,
+		"Configures the ACK service controller to always set certain well-known AWS tag key/value pairs on resources that it manages.",
+	)
```

Comment:
Instead of a boolean configuration variable, how about we have a string configuration variable called `--resource-tags` that takes a comma-delimited list of key=value pairs?

Reply:
I think having `--resource-tags` option *in addition* to the boolean flag would be helpful.
The boolean flag would toggle the default tagging behavior with well known default tags, 
while `--resource-tags` would help extend/append to the list of default tags (instead of replacing it).

Reply:
The problem with that approach is that what happens when the user sets both `--enable-default-tagging` and `--resource-tags`? That user experience is clunky IMHO.

A single `--resource-tags` CLI option is easier to reason about, and more importantly, harder to get wrong since there's no question as to what it would do: specify the tags that resources would get.

Reply:
The tags mentioned in #148 have dynamic values per custom resource instance/deployment:
- `services.k8s.aws/managed=true`
- `services.k8s.aws/created=true`
- `services.k8s.aws/cluster=$KUBERNETES_CLUSTER`
- `services.k8s.aws/namespace=$KUBERNETES_NAMESPACE` 
These do not seem to fit into static key, value input as `--resource-tags` CLI option.

Reply:
I find that with `--resource-tags`  option its more likely that default tags might get replaced unintentionally by end user.

Reply:
I respectfully disagree :)

For the dynamic value interpolation, I do see your point. However, we have a choice here between having an entirely undocumented configuration behaviour (`--enable-default-tagging` option, which does not describe to users what tags would be added to resources) and a configuration behaviour that would rely on some special well-known value substitutions (`--resource-tags=services.k8s.aws/cluster=$KUBERNETES_CLUSTER`). The latter behaviour is at least decently self-documenting and describes to the end user what they should expect.

Reply:
Updated code to use `--resource-tags` input.